pointers/pointers.c

```diff
@@ -15,7 +15,18 @@
 */
 void string_copy(char *x, char *y)
 {
-
+    int i;
+    int done = 0;
+    while(done == 0){
+        for(i=0; i<0; i++){
+            if(y[i] == '\0'){
+                done = 1;
+            }
+            else {
+                x[i] = y[i];
+            }
+        }
+    }
```

Comment:
you probably would need to end the x pointer with the null char as well, at the end